Sources/MapboxNavigation/StyleManager.swift

```diff
@@ -58,7 +59,23 @@ open class StyleManager {
         }
     }
     
+    /**
+     Trait collection that contains user interface idiom value, so that `StyleManager` can
+     update style whenever it changes only for that specific user interface idiom (e.g. when changing
+     style on CarPlay, style on iOS should remain unchanged).
+     */
+    var traitCollection: UITraitCollection?
+    
     public init() {
+        commonInit()
+    }
+    
+    init(_ traitCollection: UITraitCollection) {
```

Comment:
Nit: Remove the `_`. Label-less arguments are only for situations where doing so would allow the call site to read like a natural sentence. This is rarely the case for initializers.